Bio/PDB/FragmentMapper.py

```diff
@@ -109,30 +105,26 @@ def __init__(self, length, fid):
         self.fid = fid
 
     def get_resname_list(self):
-        """
-        @return: the residue names
-        @rtype: [string, string,...]
+        """@return: the residue names
```

Comment:
About about using ``Get residue list`` as a (better than nothing) one line summary, and leaving the parameters in the main body of the docstring as before?